### PR TITLE
adding dry run and zh system-arch pic

### PR DESCRIPTION
**Pull request by BinaryHB0916**
PTAL @hongchaodeng @wonderflow

### CODE REVIEW

**Comment by zzxwill** [comment]
What's PTAL?

File: docs/platform-engineers/debug/dry-run.md [hunk 1]
```diff
@@ -1,3 +1,107 @@
...
+      traits:
+        - type: ingress
+          properties:
+            domain: testsvc.example.com
+            http:
+              "/": 8000
+```
+
+```shell
+kubectl vela dry-run -f app.yaml
```
Comment (zzxwill) [inline docs/platform-engineers/debug/dry-run.md:33]: Why don't we use `-dry-run` or `--dry-run` for the implementation of the CLI?

File: docs/platform-engineers/debug/dry-run.md [hunk 2]
```diff
@@ -1,3 +1,107 @@
...
+      paths:
+      - backend:
+          serviceName: express-server
+          servicePort: 8000
+        path: /
+
+---
+```
+
+In this example, the definitions(`webservice` and `ingress`) which `vela-app` depends on is the built-in
```
Comment (zzxwill) [inline docs/platform-engineers/debug/dry-run.md]:
```suggestion
In this example, the definitions(`webservice` and `ingress`) which `vela-app` depends on are the built-in 
```

File: docs/platform-engineers/debug/dry-run.md [hunk 3]
```diff
@@ -1,3 +1,107 @@
...
+          servicePort: 8000
+        path: /
+
+---
+```
+
+In this example, the definitions(`webservice` and `ingress`) which `vela-app` depends on is the built-in 
+components and traits of KubeVela. You can also use `-d `or `--definitions` to specify your local definition files.
+
+`-d `or `--definitions` permitting user to provide capability definitions used in the application from local files.
```
Comment (zzxwill) [inline docs/platform-engineers/debug/dry-run.md]:
```suggestion
`-d `or `--definitions` permits users to provide capability definitions used in the application from local files.
```

File: docs/platform-engineers/debug/dry-run.md (hunk 1 above)
Comment (BinaryHB0916) [reply docs/platform-engineers/debug/dry-run.md:33]: I'm don't really know. It seems that in vela CLI there isn't any command beginning with a dash or double dash.

**Review by zzxwill — APPROVED** [review]
/lgtm

**Review by wonderflow — APPROVED** [review]

**Pull request merged**